incubator/insights-sumologic/README.md

```diff
@@ -0,0 +1,150 @@
+### NuoDB Insights and Sumo Logic
+
+---
+
+Sumo Logic is an observability suite offering end-to-end observability and security tools within one platform. Sumo Logic is an alternative monitoring solution to the current NuoDB Insights suite.
+
+#### Setup of database values yaml file
+
+---
+
+Using the NuoDB [database](https://github.com/nuodb/nuodb-helm-charts/blob/master/stable/database/README.md) helm chart, set in values file the following values for ```nuocollector.*``` section.
+
+Refer to [Sumo Logic Collectors](#sumo-logic-collectors) to harvest the Sumo Logic endpoint to insert into the yaml snippet below.
```

Comment:
Consider reordering the sections so that this is between Sumo Logic sections. Maybe something like
1. How to create a create a collector
2. Changes to make to the database helm charts
3. How to add dashboards

It does mean that the file switches between NuoDB and Sumo Logic configuration, but it is the same switch that the reader needs to make if they are following along with the guide. It avoids an issue of the reader creating dashboards, seeing no data in them, and needing to remember that they have to go back and set up the helm values.

Reply:
Agree. I've rearranged the flow to match how a user may follow the guide.